client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
This sounds reasonable but I’d prefer if there was a way to show errors/warnings from compilation but without the server logs/errors/warnings.